runtime/src/accounts_db.rs

```diff
@@ -951,6 +951,8 @@ struct LatestAccountsIndexRootsStats {
     uncleaned_roots_len: AtomicUsize,
     previous_uncleaned_roots_len: AtomicUsize,
     roots_range: AtomicU64,
+    cleaned_roots: AtomicUsize,
+    not_a_root_cleaned: AtomicUsize,
```

Comment:
how about and `rooted_cleaned_count` and `unrooted_cleaned_count` to communicate we're specifying a tally of the total cleans